### PR TITLE
Draft changes for pseudo PIP blockage data.

### PR DESCRIPTION
**Pull request by clavin-xlnx**
Minimal pseudo PIP blockage map data.

### CODE REVIEW

**Comment by clavin-xlnx** [comment]
@litghost - please take a look when it is convenient.  This is a minimal pseudo PIP blockage information map.  Perhaps there are other details you'd like to see added?

File: interchange/DeviceResources.capnp [hunk 1]
```diff
@@ -61,10 +61,11 @@ struct Device {
...
   struct TileType {
-    name      @0 : StringIdx;
-    siteTypes @1 : List(SiteTypeInTileType);
-    wires     @2 : List(StringIdx);
-    pips      @3 : List(PIP);
+    name       @0 : StringIdx;
+    siteTypes  @1 : List(SiteTypeInTileType);
+    wires      @2 : List(StringIdx);
+    pips       @3 : List(PIP);
+    pseudoPIPs @4 : List(PseudoPIP);
```
Comment (litghost) [inline interchange/DeviceResources.capnp:68]: Are all `PseudoPIP` duplicated in the `PIP` array?  Does that make sense?  Should the `PseudoPIP` just be part of `PIP` as a `union { void / pseudoCells }`?

Comment (clavin-xlnx) [reply interchange/DeviceResources.capnp]: Thanks @litghost , that is a good point.  I have moved PseudoCells under PIP combined with a union with void.

**Review by litghost — APPROVED** [review]
LGTM

**Pull request merged**